.eslintrc.json

```diff
@@ -36,7 +36,7 @@
     "prefer-const": "error",
     "curly": ["warn", "multi-line", "consistent"],
     "no-console": "off",
-    "import/extensions": ["error", "always"],
+    "import/extensions": ["error", "always", { "ignorePackages": true }],
```

Comment:
Which line complains without this change?

Reply:
this line:
https://github.com/pmndrs/jotai/blob/cc35d1d222120e401d545e4991619eb49aea97b7/vitest.config.ts#L1

Reply:
Ohhhh, because it has a slash, right? I think it's the incompleteness of the plugin.
In that case, let's add an inline comment `// eslint-disable-next-line import/extensions` and revert the change in eslintrc.